### PR TITLE
Added Health workshop card and payment option

### PR DESCRIPTION
**Pull request by Sukhamjot-Singh**
I called the API accordingly and created the card.
Please review.

### CODE REVIEW

**Comment by Sukhamjot-Singh** [comment]
This PR solves issue #11

**Comment by audreyfeldroy** [comment]
Amazing, thank you so much @Sukhamjot-Singh! Your diff looks excellent upon my initial review. I'll review it more closely as soon as the deploy preview gets created.

@pydanny when you get a chance, can you approve the deploy preview request?

File: pages/HealthWorkshopCard.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+function HealthWorkshopCard() {
+    const router = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+      const response = await fetch("/api/create-stripe-session", {
+        body: JSON.stringify({
+          amount: 500*100,
```
Comment (audreyfeldroy) [inline pages/HealthWorkshopCard.js:11]:
```suggestion
          amount: 5000*100,
```

**Review by audreyfeldroy — CHANGES_REQUESTED** [review]

**Comment by audreyfeldroy** [comment]
Merged! Nice work @Sukhamjot-Singh, hugely appreciated.

**Comment by Sukhamjot-Singh** [comment]
> Merged! Nice work @Sukhamjot-Singh, hugely appreciated.

Thank you!

**Pull request merged**